code/modules/atmospherics/pipes.dm

```diff
@@ -454,6 +459,11 @@
 	if(node3)
 		node3.disconnect(src)
 		node3 = null
+	if(in_stasis)
+		var/obj/machinery/clamp/C = locate() in get_turf(src)
+		if(C.target == src)
+			C.open()
+			C.removal()
```

Comment:
Should this block be at level `/obj/machinery/atmospherics/pipe`?

Reply:
OH SHIT THANKS YEAH

Reply:
Since it calls parent moving it to `/obj/machinery/atmospherics/pipe/Destroy()` causes no issues.